nodes/audio_utils/faster_whisper.py

```diff
@@ -0,0 +1,123 @@
+from faster_whisper import WhisperModel
+import numpy as np
+from collections import deque
+
+
+class WhisperTranscribe:
```

Comment:
@ad-astra-video this can be replaced by https://discord.com/channels/423160867534929930/1373975735110336563/1375075738499289168 and put in a sperate node repository.